geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/GMSMembership.java

```diff
@@ -652,6 +660,10 @@ public GMSMembership(MembershipListener<ID> listener, MessageListener<ID> messag
     this.listener = listener;
     this.messageListener = messageListener;
     this.gmsManager = new ManagerImpl();
+    LinkedBlockingQueue<Runnable> feed = new LinkedBlockingQueue<>();
+    ThreadFactory threadFactory = new LoggingThreadFactory("Geode View Processor");
+    this.viewExecutor = new ThreadPoolExecutor(1, 1, 30,
+        TimeUnit.SECONDS, feed, threadFactory, new ViewExecutorBlockHandler(feed));
```

Comment:
Since this is an unbounded queue, do you really need the ViewExecutorBlockHandler? Could you just use the AbortPolicy in the JDK?

Reply:
I was trying to capture the config of the old executor but I think you're right, Dan.  I'll get rid of that class and use a simpler executor.